css/amp.css

```diff
@@ -814,6 +789,30 @@ amp-accordion > section[expanded] > :last-child {
   display: block !important;
 }
 
+/* amp-list CSS to avoid FOUC */
+
+/**
+ * The default amp-hidden changes visibility, which takes up space. In the case
+ * where amp-list becomes a container via the resizable-children attribute, the
+ * loader should be fully hidden and not take up space.
+ */
+ amp-list[resizable-children] > .i-amphtml-loading-container.amp-hidden {
```

Comment:
this is does not work with the new resizable stuff. (need to fix this before launching resizable work but can be moved to amp-list.css anyway)

Reply:
Yup. I'll fix this separately referencing the amp-list dynamic resize stuff.